app/redux/ui/reducer.ts

```diff
@@ -1,14 +1,16 @@
+import StoreService from '../../../desktop/storeService';
```

Comment:
First of all,
`desktop` — is for the main process
`app` — is for the renderer
`shared` — for some things that are used in both. E.G. types, utilities.

Do not use the code of main process in the renderer and vice versa.